targets/targets.json

```diff
@@ -1780,6 +1781,7 @@
             "Freescale_EMAC",
             "PSA"
```

Comment:
We already have PSA in extra_labels, do we need `"features_add" : ["PSA"]` for K64F, K66F, UBLOX_EVK_ODIN_W2,  GD32_F450ZI and NUCLEO_F429ZI